docs/user_guide/modeling_extensions/index.rst

```diff
@@ -71,6 +72,12 @@ Estimation (MHE).
     :width: 500px
     :align: center
 
+.. rubric:: Uncertainty Propagation Toolbox
+
+:ref:`uncertainty_propagation<user_guide/modeling_extensions/uncertainty_propagation/index:Uncertainty Propagation Toolbox>`
+is a module for calculating error propagation of the objective function and constraints that include uncertain parameters to be estimated for a given model. 
```

Comment:
Alternate text: "is a module for quantifying and propagating parametric uncertainty through an optimization or simulation problem based on an IDAES model."